src/portfolios/portfolio/components/TaskOrder/TaskOrderCard.vue

```diff
@@ -99,10 +100,56 @@ import ATATMeatballMenu from "@/components/ATATMeatballMenu.vue";
 })
 export default class TaskOrderCard extends Vue {
   @Prop() private taskOrders!:TaskOrderCardData;
+  @PropSync("showDetails",{default: false}) private _showDetails!: boolean;
 
+  public menuItems(status:string):MeatballMenuItem[]
+  {
+    if(status != 'Expired') {
+      return [
+        {
+          title: "View task order details",
+          action: 'showTaskOrderDetails',
+        },
+        {
+          title: "Request to modify task order",
```

Comment:
since this is the only different menu option, I'd make a variable holding the 3 common items, then after that, if "Expired", insert this second menu option at in the 2nd position in the array... instead of having 2 different returns